### ISSUE TITLE
Zerocoin Dissapear when Attempting to Spend Back to Basecoin

### ISSUE THREAD

**Issue by CryptoSifu** [Veil-Project/veil#320]
tried spending zerocoin to basecoin using console command;
Spendzerocoin 120 false true 42 <bv address>

Zerocoins would send to destination address. 

Error: unable to find transactions containing mint (error -4)

 v1.0.0.10

 Win 10 x64 intel 4core cpu

 when the transaction failed the first time, support channel recommend I rescanzerocoinwallet and rescanblockchain. When rescanblockchain finished I tried to send the transaction again. The result was that 50% of my zerocoins dissappeared from the wallet. 
All zerocoin mints are from mined coins and are valid on chain but half are now gone. 

 working from mobile. No debug available. Atm

**Comment by seanPhill** [comment]
1. Please supply the entire command that you used to spend your coins, and then please supply the transaction ID so we can look for them on the blockchain.

2. Please also note that the Console is fussy about spelling. Even Spendzerocoin is invalid compared to spendzerocoin. Commands are case-sensitive. AND the command requires a destination address, otherwise it will spend the amount to a new _change_ address, which will not be a basecoin address.

3. The error should have been "Unable to find transaction containing mint" not "Unable to find transaction**s** containing mint". Please check your transaction ID and try ```abandontransaction "txid"```

**Comment by CryptoSifu** [comment]
1. Command used was;
spendzerocoin 120 false true 42 (my bv address)
There was no txid. Just the error. Unable to find transaction containing mint. 

2. I typed the original issue on my phone so autocorrect made those changes. Command used as above. 

3. No txid as transaction was not broadcast. 

I followed support recommendation to rescan blockchain and when it completed, attempted to use the same command to spend zerocoins. 

The new error said insufficient funds. It was at that point I noticed that half my zerocoins were gone. 

There was no txid or any indicator of where they went in the GUI display. 
I tried another rescan and searchdeterministic but they are still not there. 

Will attempt to recover from seed on fresh install to see if I can get them back. 
If I am successful I will advise.
